src/server/game/Movement/MotionMaster.cpp

```diff
@@ -253,12 +253,8 @@ void MotionMaster::MoveFollow(Unit* target, float dist, ChaseAngle angle, Moveme
     if (!target || target == _owner)
         return;
 
-    TC_LOG_DEBUG("misc", "%s (Entry: %u, GUID: %u) follows %s (Entry %u, GUID: %u).",
-        _owner->GetTypeId() == TYPEID_PLAYER ? "Player" : "Creature",
-        _owner->GetEntry(), _owner->GetGUID().GetCounter(),
-        target->GetTypeId() == TYPEID_PLAYER ? "player" : "creature",
-        target->GetEntry(), target->GetGUID().GetCounter());
-
+    TC_LOG_DEBUG("misc", "MotionMaster::MoveFollow: %s (Entry: %u, GUID: %u) follows %s (Entry %u, GUID: %u)", _owner->GetTypeId() == TYPEID_PLAYER ? "Player" : "Creature", _owner->GetEntry(), _owner->GetGUID().GetCounter(),
```

Comment:
ObjectGuid::ToString can simplify this log message

Reply:
just did some formating no real change, but I can give it a look

Reply:
I really want people to use ObjectGuid::ToString instead of manually trying to print its pieces...